cmd/pitr/collector/collector.go

```diff
@@ -219,7 +234,11 @@ func (c *Collector) CollectBinLogs() error {
 		}
 
 		if lastUploadedBinlogName == "" {
-			log.Println("Gap detected in the binary logs. Binary logs will be uploaded anyway, but full backup needed for consistent recovery.")
+			log.Println("ERROR: Couldn't find the binlog that contains GTID set:", c.lastSet)
+			log.Println("ERROR: Gap detected in the binary logs. Binary logs will be uploaded anyway, but full backup needed for consistent recovery.")
+			if err := createGapFile(c.lastSet); err != nil {
+				return errors.Wrap(err, "create gap file")
```

Comment:
Could we put GTID info in this message too? Just make it easier to check.

Reply:
I don't understand the value to add the GTID into this error. The error will happen only if the file is failed to create, how GTID will make it easier to troubleshoot?